[21812 ] Add new verbose modes and search topics using wildcards (#99)
eProsima/Fast-DDS-spy: docs/rst/user_manual/commands/topic.rst

Arguments
=========

> **Topic** command support from 0 to 2 arguments:

Reviewer: ```suggestion
**Topic** command supports from 0 to 2 arguments:
```

Verbose
-------

This argument queries for more complete information about each of the topics in the network.

> It adds the Guid of each endpoint on the topic and the whether the type has been discovered.

Reviewer: Why was this removed? Still holds right?

Reviewer 2: The guid is only for the higher verbosity mode

Reviewer: So we are only documenting what information is provided through examples? Maybe it should be explained what extra information is showed in verbose mode, and in super verbose mode.

    If there are 2 topics with the same name and different Topic Data Type, only one of them could be visible.
    These is a circumstance that :term:`DDS` allows, but it is strongly suggested not to do.

Topic name with wildcards
-------------------------

> When a topic name is with wildcards (*), this command retrieves :ref:`verbose information <user_manual_command_topic_output_verbose>` for all topics that matches the given filter.

Reviewer: ```suggestion
When a topic name contains wildcards (*), this command retrieves :ref:`verbose information <user_manual_command_topic_output_verbose>` for all topics that match the given filter.
```
